clustered/client/build.gradle

```diff
@@ -17,24 +17,29 @@
 apply plugin: EhDeploy
 
 dependencies {
-  compileOnly project(':api')
+  compileOnly project(':impl')
```

Comment:
Is it because we know here that we can access everything both in impl and api because the shaded jar at the end will contain both ?

Reply:
we get `:api` transitively through `:impl`... it's redundant.